### DISCUSSION TITLE
Reauth event in vici interface

### DISCUSSION THREAD

**Discussion by nknkgithub** [strongswan/strongswan#2255] [Q&A]
Do we have support for reauth event in vici interface?
Like the events related to child-updown as documented in below link.
https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md

How do we distinguish between reauth and child-updown/ike-updown through vici callbacks?

**Comment by nknkgithub** [comment]
Help required!!

**Comment by tobiasbrunner** [comment]
> How do we distinguish between reauth and child-updown/ike-updown through vici callbacks?

You can't really.
